x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MlAnomaliesIndexUpdate.java

```diff
@@ -163,9 +175,18 @@ private void rollAndUpdateAliases(ClusterState clusterState, String index, Actio
     }
 
     private void rollover(String alias, @Nullable String newIndexName, ActionListener<String> listener) {
-        client.admin().indices().rolloverIndex(new RolloverRequest(alias, newIndexName), listener.delegateFailure((l, response) -> {
-            l.onResponse(response.getNewIndex());
-        }));
+        client.admin()
+            .indices()
+            .rolloverIndex(
+                new RolloverRequest(alias, newIndexName),
+                ActionListener.wrap(response -> listener.onResponse(response.getNewIndex()), e -> {
```

Comment:
Maybe you can add a comment explaining why we ignore the `ResourceAlreadyExistsException`.